src/main/diagnostics/rdBinInShell.ts

```diff
@@ -39,12 +40,15 @@ class RDBinInShellPath implements DiagnosticsChecker {
     let description: string;
 
     try {
-      const { stdout } = await spawnFile(this.executable, this.args, { stdio: 'pipe' });
-      const dirs = stdout.trim().split(/[:\n]/);
+      const { stdout } = await spawnFile(this.executable, this.args, { stdio: ['ignore', 'pipe', 'pipe'] });
+      const outputLines = stdout.split('\n').filter(line => line.startsWith(pathOutputDelimiter));
+      const dirs = outputLines[outputLines.length - 1].substring(pathOutputDelimiter.length).split(':');
```

Comment:
```suggestion
      const dirs = outputLines.pop().substring(pathOutputDelimiter.length).split(':');
```
Unless you wanted to keep `outputLines` unchanged for some reason.

Reply:
If we make `pathOutputDelimiter` end with `:`, then we can just skip the `substring` call (because the prefix will not be the correct path).

Reply:
Both good points. Funny I was wanting ruby/python's `ARRAY[-1]` and forgot js's `pop()`.